.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: yes
+
+contact_links:
```

Comment:
These are spaces that are quite specific to the _tooling_ that's used to build TSPL, rather than discussion about TSPL in general. It feels like we should create a TSPL or "Swift Book" category in https://forums.swift.org/c/swift-documentation/92 and link it from here.

Reply:
At this stage, we're focused on TSPL work that supports moving to DocC — and most of that will be either making tooling changes or making content changes to adopt that tooling work.  Do you think there's value in separating those discussions?  I think creating a separate area for documentation might be more useful later, when TSPL work is focused on content changes.  When we create that area, we should make it general enough to include discussions around Swift standard library docs and other developer-facing documentation.

Reply:
I don't mind either approach. Having discussions in a new 'Swift Documentation' category now will help inform future contributors where discussions related to Swift Documentation should go.

Reply:
I added separate links in 7dd66dd5.